marlowe-cli/cli-test/Language/Marlowe/CLI/Test/Contract/ParametrizedMarloweJSON.hs

```diff
@@ -60,25 +67,58 @@ instance FromJSON ParametrizedMarloweJSON where
 instance ToJSON ParametrizedMarloweJSON where
   toJSON (ParametrizedMarloweJSON json) = json
 
-rewriteCurrencyRefs :: Currencies -> ParametrizedMarloweJSON -> Either CurrencyNickname ParametrizedMarloweJSON
-rewriteCurrencyRefs (Currencies currencies) (ParametrizedMarloweJSON json) = ParametrizedMarloweJSON <$> A.rewriteBottomUp rewrite json
+-- It can be:
+
+-- * a pair (A.Array [A.String "", A.String ""])
+
+-- * a pair (A.Array [A.String currencyNickname, A.String tokenName])
+
+-- * (A.String "ADA")
+
+-- * (A.Object [("currency_symbol", A.String ""), ("token_name", A.String "")])
+
+-- * (A.Object [("currency_symbol", A.String currencyNickname), ("token_name", A.String tokenName)])
```

Comment:
We're going to need to write a user manual for the testing DSL soon.

Reply:
For sure. We can discuss how we would like to approach this - I have few additions (custom script integration etc.).